src/auth/computeclient.ts

```diff
@@ -68,8 +74,11 @@ export class Compute extends OAuth2Client {
     try {
       // TODO: In 2.0, we should remove the ability to configure the tokenUrl,
       // and switch this over to use the gcp-metadata package instead.
-      res = await this.transporter.request<CredentialRequest>(
```

Comment:
So is `this.transporter` no longer used?

Reply:
It's used in other places, just not in this particular case.  All that transporter thing really does is attach user agent headers.   This is important in places like the apiary client, but not relevant in this case.